packages/visualizations/addon/components/cell-renderers/dimension.js

```diff
@@ -10,14 +10,63 @@
  * }}
  */
 
-import Ember from 'ember';
+import Component from '@ember/component';
+import { computed, get } from '@ember/object'
 import layout from '../../templates/components/cell-renderers/dimension';
 
-export default Ember.Component.extend({
+export default Component.extend({
   layout,
 
   /**
    * @property {Array} classNames - list of component class names
    */
-  classNames: ['table-cell-content', 'dimension']
+  classNames: ['table-cell-content', 'dimension'],
+
+  /**
+   * @property {String} descField - field to find the description
+   */
+  descField: computed('column.field.dimension', function() {
+    return get(this, 'column.field.dimension') + '|desc';
+  }),
+
+  /**
+   * @property {String} idField - field to find id
+   */
+  idField: computed('column.field.dimension', function() {
+    return get(this, 'column.field.dimension') + '|id';
+  }),
+
+  /**
+   * @property {String} title - value that should be used in hoverover title
+   */
+  title: computed('descField', 'idField', 'data', function() {
+    let descField = get(this, 'descField'),
+        idField = get(this, 'idField'),
+        data = get(this, 'data');
+
+    if(data[idField]) {
+      return data[idField];
+    } else if(data[descField]) {
+      return data[descField];
+    }
+
+    return '--';
+  }),
+
+  /**
+   * @property {String} value - value that should be displayed in table cell
+   */
+  value: computed('descField', 'idField', 'data', function() {
```

Comment:
Is this the exact same as `title`?

Reply:
Do we already have tests for these changes?

Reply:
display priority is different.

Tests added for titles